### PR TITLE
Comments for the Components

### PR DESCRIPTION
**Pull request by jrcarretas**

### CODE REVIEW

File: frontend/components/arts/ArtistCard.vue [hunk 1]
```diff
@@ -80,34 +82,49 @@ export default {
...
+      // Nice -> Could be renamed to mediaExists. With 's' as it is singular.
       return this.mediaData.media_file !== null
     },
     artImage() {
+      // Good usage. Would be better if this had a comment. It's not readable, so a helptext would be great.
       return this.mediaExist
         ? this.mediaData.media_file
         : this.mediaType === 'video'
         ? this.videoThumbnail
         : require('@/assets/images/public_art_icon.svg')
```
Comment (countable) [inline frontend/components/arts/ArtistCard.vue:122]: How do you guys feel about ternary ops versus if {} else {} blocks?

I'd agree ternary ops have some advantage when returning a value

File: frontend/components/arts/ArtistCard.vue [hunk 2]
```diff
@@ -120,6 +137,9 @@ export default {
       return `By ${listOfArtist}`
     },
     returnMediaType() {
+      // This is a bit weird. You're not returning a mediaType, so this shouldn't be
+      // named as returnMediaType or simply mediaType. You're returning a PNG, so this
+      // should be something else. `mediaTypeImage`?
       let mediaType = ''
```
Comment (countable) [inline frontend/components/arts/ArtistCard.vue:143]: This function seems to do nothing. Cant  the callee just call doneReveal directly?

File: frontend/components/arts/ArtistCard.vue (hunk 1 above)
Comment (jrcarretas) [reply frontend/components/arts/ArtistCard.vue:122]: For me, if it's simple, I'd use the ternary operator. If it's not, I'd use an if-else block.

File: frontend/components/arts/ArtistCard.vue (hunk 2 above)
Comment (jrcarretas) [reply frontend/components/arts/ArtistCard.vue:143]: doneReveal is from a different function. Github awkwardly sliced this part. Here's the rest of the current function.
image

**Comment by countable** [comment]
Guys what's the plan for this branch. @jrcarretas ?

**Comment by jrcarretas** [comment]
I'm keeping it for now since we haven't fixed everything pointed out in
this branch. For reference purposes.

**Pull request closed without merging**